PWGHF/Tasks/HFMCValidation.cxx

```diff
@@ -350,14 +358,24 @@ struct ValidationRecLevel {
   std::array<std::array<std::shared_ptr<TH1>, 2>, nCharmHadrons> histPtReco;
   std::array<std::shared_ptr<THnSparse>, 4> histOriginTracks;
   std::shared_ptr<TH2> histAmbiguousTracks, histTracks;
+  std::shared_ptr<TH1> histContributors;
+
+  AxisSpec axisDeltaMom{2000, -1., 1.};
+  AxisSpec axisOrigin{4, -1.5, 2.5};
+  AxisSpec axisEta{40, -1., 1.};
+  AxisSpec axisPt{50, 0., 10.};
+  AxisSpec axisDeltaVtx{200, -1, 1.};
+  AxisSpec axisDecision{2, -0.5, 1.5};
+  AxisSpec axisITShits{8, -0.5, 7.5};
 
   HistogramRegistry registry{
     "registry",
-    {{"histXvtxReco", "Position of reco PV in #it{X};#it{X}^{reco} (cm);entries", {HistType::kTH1F, {{200, -1, 1.}}}},
-     {"histYvtxReco", "Position of reco PV in #it{Y};#it{Y}^{reco} (cm);entries", {HistType::kTH1F, {{200, -1, 1.}}}},
+    {{"histNtracks", "Position of reco PV in #it{X};#it{X}^{reco} (cm);entries", {HistType::kTH1F, {{200, 0., 200.}}}},
```

Comment:
Looks like forgotten copy-paste.

Reply:
Thanks for spotting it! Indeed it was a copy-paste leftover, I just fixed it